nixpkgs/github.json

```diff
@@ -3,6 +3,6 @@
   "repo": "nixpkgs",
   "branch": "reflex-platform-20.09",
   "private": false,
-  "rev": "a78062dc78a85f5d553443ea636a3242c0e6de04",
-  "sha256": "1kda0hdmvscma86c13z01mm29ka43djd9fskfa9g6f170sd9f4qc"
+  "rev": "174ae1100165812aea5433dc66549244bda917d1",
+  "sha256": "1h8pgf19m6j1l2cwvfg19ikxklyvcqccrc1i0m2lkvcilvjdzgqf"
```

Comment:
This was a very small workaround that should not be user-visible except for the fact that it fixes eval.